com.wps.WPS.json

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "com.wps.WPS",
+    "branch": "alpha",
```

Comment:
Should be `stable`.

Reply:
Is that compulsory to have the default branch as 'stable'? Since upstream has no a 'stable' branch yet. Upstream has only 'alpha' and 'beta' branches.

Reply:
I get from the build result. Then, should I blindly set the branch to stable, just for flathub?